packages/binding-netconf/src/xpath2json.ts

```diff
@@ -12,7 +12,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR W3C-20150513
  ********************************************************************************/
-var util = require("util");
+const util = require("util");
```

Comment:
Can't we use `import` here?

Reply:
I noticed that the import can actually be removed as `util` isn't used anywhere :) I will push an updated commit soon.

Reply:
I covered the change :)